### DISCUSSION TITLE
graphql 개발 셋팅

### DISCUSSION THREAD

**Discussion by develjsh** [lama-org/semi-back#9] [Q&A]
우석님 질문 있습니다~~
우석님이 링크 걸어준곳 읽던 중 이런 코드가 있는데 궁금한 점이 있어서요.

const resolvers = {
  Query: {
    posts: () => posts,
    author: (_, { id }) => find(authors, { id }),
  },
}

여가에 author 하고    __,(언더라인 & 코마) 이렇게 되어 있는데, 이건 어떤 의미 인건가요?

**Comment by dntjr031** [comment]
> 우석님 질문 있습니다~~
> 
> 우석님이 링크 걸어준곳 읽던 중 이런 코드가 있는데 궁금한 점이 있어서요.
> 
> 
> 
> const resolvers = {
> 
>   Query: {
> 
>     posts: () => posts,
> 
>     author: (_, { id }) => find(authors, { id }),
> 
>   },
> 
> }
> 
> 
> 
> 여가에 author 하고    __,(언더라인 & 코마) 이렇게 되어 있는데, 이건 어떤 의미 인건가요?

앗 지금 확인했네요 ㅠㅠ
사용하지 않기때문에 아무 의미없는 별칭으로 적어놓은 것입니다!
